.teamcity/Gradle_Check/configurations/BuildDistributions.kt

```diff
@@ -1,11 +1,12 @@
 package configurations
 
+import jetbrains.buildServer.configs.kotlin.v2018_1.AbsoluteId
 import model.CIBuildModel
 import model.Stage
 
 class BuildDistributions(model: CIBuildModel, stage: Stage) : BaseGradleBuildType(model, stage = stage, init = {
     uuid = "${model.projectPrefix}BuildDistributions"
```

Comment:
Looking at the KDoc for `id`, we could get rid of setting `uuid`, since the value of ID will be used anyway. WDYT? Maybe in a follow up PR?

Reply:
I actually tried to remove the UUID but then the `mvn org.jetbrains.teamcity:teamcity-configs-maven-plugin:2018.1:generate` command gave me validation errors. I'd postpone the removal to a separate PR.